src/components/app/ProfileName.js

```diff
@@ -118,34 +118,41 @@ class ProfileNameImpl extends React.PureComponent<Props, State> {
     // controlled by CSS.
     return (
       <>
-        <button
-          type="button"
-          style={{
-            display: isFocused ? 'none' : null,
-          }}
-          title={title}
-          className="profileNameButton menuButtonsButton menuButtonsButton-hasRightBorder menuButtonsButton-hasIcon"
-          onFocus={this.handleButtonFocus}
-          onClick={this.handleButtonFocus}
+        <Localized id="ProfileName--profile-name" attrs={{ title: true }}>
```

Comment:
I believe it would be better to have 2 different ids for the 2 different elements.

What do you think of:

`ProfileName--edit-profile-name-button`
and
`ProfileName--edit-profile-name-input`

?

Reply:
Done. 
I did git commit --amend and  git push --force my branch so i do not have the very latest commits.
Shall i add them? If yes, is rebasing main to my branch the way or do i need a merge commit or sth similar?

Reply:
Thanks!
Usually we use rebase in this project, but that's not really necessary unless there are conflicts (github should tell when it's happening).
Before merging we do a merge directly with the github interface to get the tests running on the latest code, and then merge from there.